lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_publication
+short_description: Add or remove PostgreSQL publication from a remote host
+description:
+   - Add or remove PostgreSQL publication from a remote host.
+version_added: "2.5"
+options:
+  name:
+    description:
+      - Name of the publication to add or remove.
+    required: true
+    default: null
+  database:
+    description:
+      - Name of the database to connect to.
+    required: false
+    default: postgres
+  tables:
+    description:
+      - List of the tables to target for publication.
+      - If not value is set all tables are targeted.
+    required: false
+    default: null
+  state:
+    description:
+      - The publication state.
+    required: false
+    default: present
+    choices: [ "present", "absent" ]
+notes:
+   - This module uses I(psycopg2), a Python PostgreSQL database adapter. You must ensure that psycopg2 is installed on
+     the host before using this module. If the remote host is the PostgreSQL server (which is the default case),
+     then PostgreSQL must also be installed on the remote host. For Ubuntu-based systems, install the C(postgresql),
+     C(libpq-dev), and C(python-psycopg2) packages on the remote host before
+     using this module.
+   - PostgreSQL version should be C(10.0) or greater.
+requirements: [ psycopg2 ]
+author: "Loic Blot <loic.blot@unix-experience.fr>"
```

Comment:
Use `First Last (@GitHubID)` for `author` ([see](http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block))